projects/plugins/protect/src/js/components/firewall-footer/index.jsx

```diff
@@ -0,0 +1,71 @@
+import { AdminSectionHero, Title, Text, Button } from '@automattic/jetpack-components';
+import { CheckboxControl } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { STORE_ID } from '../../state/store';
+import SeventyFiveLayout from '../seventy-five-layout';
+import styles from './styles.module.scss';
+
+const StandaloneMode = () => {
+	const { setModal } = useDispatch( STORE_ID );
+
+	const handleClick = () => {
+		return event => {
+			event.preventDefault();
+			setModal( {
+				type: 'STANDALONE_MODE',
+			} );
+		};
+	};
+
+	return (
+		<div className={ styles[ 'standalone-mode-section' ] }>
+			<Title>{ __( 'Standalone mode', 'jetpack-protect' ) }</Title>
+			<Text mb={ 3 }>
+				{ __(
+					'Learn how you can execute the firewall before WordPress initializes. This mode offers the most protection.',
+					'jetpack-protect'
+				) }
+			</Text>
+			<Button
+				variant={ 'link' }
+				isExternalLink={ true }
+				weight={ 'regular' }
+				onClick={ handleClick() }
+			>
+				{ __( 'Learn more', 'jetpack-protect' ) }
+			</Button>
+		</div>
+	);
+};
+
+const ShareData = () => {
+	return (
+		<div className={ styles[ 'share-data-section' ] }>
+			<Title>{ __( ' Share data with Jetpack', 'jetpack-protect' ) }</Title>
+			<div className={ styles[ 'footer-checkbox' ] }>
+				<CheckboxControl checked={ true } />
+				<Text>
+					{ __(
+						'Allow Jetpack to collect data to improve firewall protection and rules. Collected data is also used to display advanced usage metrics.',
+						'jetpack-protect'
+					) }
+				</Text>
+			</div>
+		</div>
+	);
+};
+
+const FirewallFooter = () => {
+	return (
+		<AdminSectionHero>
+			<SeventyFiveLayout
```

Comment:
We need `56px` of vertical spacing between the two columns on mobile, I believe you can check out `ScanFooter` to see where that is done!

<img width="252" alt="Screen Shot 2022-11-22 at 12 32 34 PM" src="https://user-images.githubusercontent.com/10933065/203404919-15621c65-d3af-48a2-96b3-95db36d0e1e6.png">